crates/sui-graphql-rpc/src/config.rs

```diff
@@ -27,6 +27,7 @@ const MAX_TYPE_NODES: u32 = 256;
 const MAX_MOVE_VALUE_DEPTH: u32 = 128;
 
 pub(crate) const DEFAULT_REQUEST_TIMEOUT_MS: u64 = 40_000;
+pub(crate) const DEFAULT_MUTATION_TIMEOUT_MS: u64 = 60_000;
```

Comment:
Can we annotate this with a comment pointing to where we derived this default from? It should be the same as the default timeout for quorum driving by fullnode (plus some margin of error).

Reply:
Added. I think this is the correct one, but if not, please let me know!

Reply:
@amnn this is the only remaining issue. Not sure what would be the right small margin of error? 10% more? 5% more?

Reply:
Thanks @stefan-mysten,

- Note that at your link, there are two timeouts there: pre-quorum of 60s and post-quorum of 7 seconds. Both of those timeouts will apply, so we should sum both of them together.
- Let's use a permalink (a link that references the exact commit) otherwise the line references will stop making sense over time.
- an extra 10% sounds fine for the margin of error to start with, and we'll see how it goes from there.

Reply:
Tip top, will do!